config/linux-parrot_common.config

```diff
@@ -0,0 +1,304 @@
+CONFIG_LOCALVERSION="-heads"
+# CONFIG_LOCALVERSION_AUTO is not set
+CONFIG_KERNEL_XZ=y
+# CONFIG_SWAP is not set
+# CONFIG_CROSS_MEMORY_ATTACH is not set
+CONFIG_NO_HZ_IDLE=y
+CONFIG_PREEMPT_VOLUNTARY=y
+CONFIG_LOG_BUF_SHIFT=18
+CONFIG_BLK_DEV_INITRD=y
+CONFIG_INITRAMFS_SOURCE="../../../blobs/dev.cpio"
+# CONFIG_RD_GZIP is not set
+# CONFIG_RD_BZIP2 is not set
+# CONFIG_RD_LZMA is not set
+# CONFIG_RD_LZO is not set
```

Comment:
All that file was created with the output of make savedefconfig in linux build dir or not?
If not please to and overwrite

Reply:
i think this is a copy from linux-librem_common.config with +CONFIG_TCG_TIS_I2C_NUVOTON=y
but i dont think that option is needed, this machine needs other modules for networking ill create a new kernel config for it tomorrow

Reply:
new version is created with savedefconfig, and has eth0 working